Alexandra/style.css

```diff
@@ -0,0 +1,104 @@
+
+
+body{
+    display: flex;
+    flex-direction: column;
+    background-color: rgb(166, 191, 127);
+    
+}
+
+.imagine{
+    width: 350px;
+    height: 250px;
+    object-fit:fill;
+  }
+
+.imagine, a{
+    color:green;
+}
+.row{
+display: flex;
+flex-direction: row;
+
+

```

Comment:
why so many extra spaces? Also flex-direction by default has the row. so if you use just flex, you will have the same input. I bet if you remove it, it will work the same way